### PR TITLE
Translating section 1 lesson 4 to PTBR

### PR DESCRIPTION
**Pull request by Lorenzobattistela**
I've just translated the last session 1 lesson. I have a doubt: what do you think of writing down credits for translating it? Maybe something like: Translation to PTBR by Lorenzo Battistela and Jmallone, what do you think?

### CODE REVIEW

File: NFT_Collection/pt-br/Section_1/Lesson_4_Create_A_Contract_That_Mints_NFTs.md [hunk 1]
```diff
@@ -1 +1,353 @@
...
+
+## 💳 Transações
+
+Então, quando nós quisermos performar uma ação que mude a blockchain, nós chamamos isso de  *transaction*. Por exemplo, mandar ETH para alguém é uma transação porque estamos mudando o saldo das contas. Fazer algo que atualiza uma variável no nosso contrato também é considerado uma transação porque estamos mudando dados. Mintar uma NFT é uma transação porque estamos salvando dados no contrato.
+
+**Implementando (deploying) um contrato inteligente também é uma transação.**
+
+Lembre-se, a blockchain não tem dono. É só um monte de computadores ao redor do mundo rodando através de **mineradores** que tem a cópia da blockchain.
+
+Quando implementarmos nosso contrato,  nós preciamos falar  **para todos esses** mineradores, "ei, esse é um contrato inteligente novo, por favor adicione meu contrato inteligente à blockchain e diga para todo mundo sobre ele também".
```
Comment (Jmallone) [inline NFT_Collection/pt-br/Section_1/Lesson_4_Create_A_Contract_That_Mints_NFTs.md]: **preciamos** -> precisamos

File: NFT_Collection/pt-br/Section_1/Lesson_4_Create_A_Contract_That_Mints_NFTs.md [hunk 2]
```diff
@@ -1 +1,353 @@
...
+    },
+  },
+};
+```
+
+Você pode conseguir URL da sua API no dashboard do Alchemy e colar ali mesmo. Depois, você vai precisar da sua chave **privada** do rinkeby (não o seu endereço público!) o qual você pode [pegar no metamask](https://metamask.zendesk.com/hc/en-us/articles/360015289632-How-to-Export-an-Account-Private-Key) e colar ali também.
+
+**Nota: NÃO FAÇA COMMIT DESSE ARQUIVO NO GITHUB. ELE CONTÉM SUA CHAVE PRIVADA. VOCÊ PODE SER ROUBADO E HACKEADO. ESSA CHAVE PRIVADA É A MESMA QUE A DA MAINNET.** Nós vamos falar sobre variáveis `.env` depois e como mantê-las em segredo.
+
+Por quê você precisa dessa chave privada? Porque para perfomar uma transação como fazer deploy de um contrato, você precisa "logar" na blockchain e assinar/ fazer deploy do contrato. E, o seu nome de usuário é o seu endereço público, e sua senha é sua chave privada. É como fazer login na AWS ou GCP para fazer deploy.
```
Comment (Jmallone) [inline NFT_Collection/pt-br/Section_1/Lesson_4_Create_A_Contract_That_Mints_NFTs.md]: **perfomar** - > performar

File: NFT_Collection/pt-br/Section_2/Lesson_1_What_Is_On_Chain_Data.md [hunk 3]
```diff
@@ -1 +1,61 @@
...
\ No newline at end of file
+## 🔗 O que "on-chain" significa e porque é importante.
+
+Nesse momento, nós temos um grande problema com nossas NFTs.
+
+O que acontece se o imgur cair? Então - nosso link da imagem seria absolutamente inútil, e nossa NFT e o Spongebob estarão perdidos! E pior, o que acontece se o site que hospeda o arquivo JSON cair? Então - nossa NFT estaria completamente quebrada pois os metdadados não seriam acessíveis.
+
+Uma maneira de consertar esse problema é armazenar os dados da nossa NFT "on-chain", significando que os dados vivem no contrado ao invés de nas mãos de um terceiro (third-party). Isso significa que nossa NFT será verdadeiramente permanente :). Nesse caso, a única situação onde perdemos nossos dados da NFT seria se a blockchain caísse. E se isso acontecer - bom, aí temos problemas maiores.
+
+Mas, assumindo que a blockchain não caia, nossa NFT vai estar ali pra sempre! Isso é bem atraente, porque também significa que se você vender uma NFT, o comprador pode estar confiante que ela não vai quebrar. Muitos projetos populares usam dados "On-chain", [Loot](https://techcrunch.com/2021/09/03/loot-games-the-crypto-world/) é um exemplo muito popular!
```
Comment (Jmallone) [inline NFT_Collection/pt-br/Section_2/Lesson_1_What_Is_On_Chain_Data.md]: **pra** -> para

File: NFT_Collection/pt-br/Section_2/Lesson_1_What_Is_On_Chain_Data.md [hunk 4]
```diff
@@ -1 +1,61 @@
...
+
+🖼 O que são SVGs?
+
+---
+
+Uma maneira comum de armazenar dados NFT para imagens é usando um SVG. Um SVG é uma imagem, mas a imagem é construída com código.
+
+Por exemplo, aqui está um exemplo simples de SVG que renderiza uma caixa prete com algum texto branco no meio.
```
Comment (Jmallone) [inline NFT_Collection/pt-br/Section_2/Lesson_1_What_Is_On_Chain_Data.md]: **prete** -> preta

File: NFT_Collection/pt-br/Section_2/Lesson_2_Minting_NFTs_With_Static_SVGs.md [hunk 5]
```diff
@@ -1 +1,127 @@
...
+    class="base"
+    dominant-baseline="middle"
+    text-anchor="middle"
+  >
+    EpicLordHamburger
+  </text>
+</svg>
+```
+
+Depois, precisamos de algum jeito pegear os dados das nossas NFT sem hospedar em algum lugar como o imgur (que pode cair ou morrer a qualquer momento!). Vá até [esse](https://www.utilities-online.info/base64) site. Cole todo o código SVG acima e clique em "encode" para pegar seu SVG base 64 encoded. Agora, está pronto para alguma mágica? Abra uma nova guia. E no URL, digite isso:
```
Comment (Jmallone) [inline NFT_Collection/pt-br/Section_2/Lesson_2_Minting_NFTs_With_Static_SVGs.md]: pegear -> pegar

File: NFT_Collection/pt-br/Section_2/Lesson_2_Minting_NFTs_With_Static_SVGs.md [hunk 6]
```diff
@@ -1 +1,127 @@
...
+
+Ok, **épico**. Essa é uma maneira de manter os dados da nossa imagem NFT permanentes e disponíveis para sempre. Todos os centros de dados do mundo podem queimar, e porque nós temos a string codificada com base64, nós sempre vamos poder ver o SVG enquanto tivermos um computador e um browser.
+
+![Untitled](https://i.imgur.com/f9mXVSb.png)
+
+☠️ Nos livrando do JSON hospedado.
+
+Lembra dos metadados JSON?
+
+Então, eu mudei um pouco para nossas NFTs de três palavras :). Mesma coisa! Um nome, descrição e imagem. Mas agora, aoa invés de apontar para um link imgur, vamos apontar pra nossa string codificada com base64.
```
Comment (Jmallone) [inline NFT_Collection/pt-br/Section_2/Lesson_2_Minting_NFTs_With_Static_SVGs.md]: **aoa** -> ao
**pra** -> para

**Review by Jmallone — CHANGES_REQUESTED** [review]
It's interesting writing our names, maybe in a **README.md** file inside the `NFT_Collection/pt-br `folder, what do you think? 

I wrote down some wrong words, done that the PR will be accepted :)

**Comment by Lorenzobattistela** [comment]
> It's interesting writing our names, maybe in a **README.md** file inside the `NFT_Collection/pt-br `folder, what do you think?
> 
> I wrote down some wrong words, done that the PR will be accepted :)


Yeah, I guess a README would be a good way, we can build a PR pattern too for contributions and also give credit for contributors.

**Review by Jmallone — APPROVED** [review]
Okay i will create a README.md or CONTRIBUTORS.md later

Already fine to me, :) +1 PR complete

**Comment by Lorenzobattistela** [comment]
> Okay i will create a README.md or CONTRIBUTORS.md later
> 
> Already fine to me, :) +1 PR complete

Maybe update the contributors file? IDK if you pretend to make a PR for the main repo, but anyway, i've just saw that that is a contributors file.

**Pull request merged**